paddle/fluid/operators/conv_mkldnn_op.cc

```diff
@@ -68,13 +69,222 @@ inline mkldnn::memory::format GetWeightsFormat(mkldnn::memory::format format,
   }
 }
 
-template <typename T>
+template <typename T, typename K>
 class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
  public:
   void Compute(const paddle::framework::ExecutionContext& ctx) const override {
     PADDLE_ENFORCE(paddle::platform::is_cpu_place(ctx.GetPlace()),
                    "It must use CPUPlace.");
 
+    bool is_INT8 =
+        std::is_same<T, int8_t>::value || std::is_same<T, uint8_t>::value;
+    if (is_INT8) {
+      ComputeINT8(ctx);
+    } else {
```

Comment:
Since there is `ComputeINT8` function, you can create `ComputeFP32` function, to make code more cleaner.
And Since `FP32` is used more frequently than `int8`
```
if (!is_INT8) {
   ComputeFP32(ctx);
} else {
   ComputeINT8(ctx);
}
```